deploy/scripts/src/accounts/test.ts

```diff
@@ -178,6 +178,9 @@ export function changeEmail(): void {
       isStatusCode200,
       ...pageContentCheck('API Simulation Tool')
     })
+    if (!pageContentCheck('API Simulation Tool').validatePageContent(res)) {
```

Comment:
I think this will never be called as this check needs to happen before the `timeGroup` checks which call `fail` if the checks fail. 
Like this
```suggestion
      res = timeGroup(groups[2].split('::')[1], () => {
      const r = http.get(res.headers.Location)
      if (!pageContentCheck('API Simulation Tool').validatePageContent(r)) {
        console.log(r.html('h1').first().text())
      }
      return r
    }, {
      isStatusCode200,
      ...pageContentCheck('API Simulation Tool')
    })
    
```